src/test/java/com/andrei/curs/validator/CarValidatorTest.java

```diff
@@ -0,0 +1,257 @@
+package com.andrei.curs.validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.math.BigDecimal;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import com.andrei.curs.exception.CarValidatorException;
+import com.andrei.curs.model.Car;
+
+@SpringBootTest
+@ActiveProfiles("local")
+public class CarValidatorTest {
+
+    Car createCustomCar(String fieldName, String fieldValueString, Integer fieldValueInteger) {
```

Comment:
not cool, duplicate lines :(
you can put everything as parameters and call it multiple times instead of this spaghetti with if/else

Reply:
I've changed the function, no more spaghetti:)